test/unit/lib/plugins/aws/package/compile/layers.test.js

```diff
@@ -393,33 +128,96 @@ describe('lib/plugins/aws/package/compile/layers/index.test.js', () => {
         awsRequestStubMap,
       });
       expect(firstCfResources).to.have.property(firstLayerResourceName);
-      // Replaces
-      // https://github.com/serverless/serverless/blob/e78f695004bf292c4163daf9705e5e0c6cbe2592/test/unit/lib/plugins/aws/package/compile/layers/index.test.js#L147-L329
     });
   });
 
-  it.skip('TODO: should generate expected permissions resource', () => {
-    // Replaces
-    // https://github.com/serverless/serverless/blob/e78f695004bf292c4163daf9705e5e0c6cbe2592/test/unit/lib/plugins/aws/package/compile/layers/index.test.js#L331-L383
+  it('should generate expected permissions resource', () => {
+    const layerResourceName = naming.getLambdaLayerLogicalId('LayerOne');
```

Comment:
let's only validate that we have the expected permission resource as a part of this test as the assertions about `layer` type, etc are duplicated with the tests above